llpc/test/shaderdb/TestPatchBufferOp.comp

```diff
@@ -0,0 +1,44 @@
+#version 450
+
+layout(local_size_x = 1, local_size_y = 1, local_size_z = 1) in;
+
+struct s
+{
+    int n;
+};
+
+layout(binding = 0) buffer _1
+{
+    s a[];
+} buffer1;
+
+layout(binding = 1) buffer _2
+{
+    int b[];
+} buffer2;
+
+void main()
+{
+    int i = buffer2.b[0];
+
+    for (;;)
+    {
+        s struct0 = buffer1.a[i];
+
+        if (i == 0)
+        {
+            break;
+        }
+
+        i = struct0.n;
+    }
+}
+
+
+// BEGIN_SHADERTEST
+/*
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+; SHADERTEST: AMDLLPC SUCCESS
+*/
+// END_SHADERTEST
```

Comment:
Could you move it to the top of the file? I find it easier to figure out what a test is doing or why it's failing when RUN lines are towards the top.
Summarizing what a test is supposed to exercise in plain English is also a good practice.

Reply:
Sure, all comments are addressed, thanks for them.